governance.md

```diff
@@ -0,0 +1,109 @@
+# Principles
+
+The Confidential Containers is an open source organization and community
+that adheres to the following principles:
+
+* Open - Confidential Containers is an open source project. We welcome all [contributions](https://github.com/confidential-containers/community/blob/main/CONTRIBUTING.md)
+* Respectful and welcoming - See our [Code of Conduct](https://github.com/confidential-containers/community/blob/main/CODE_OF_CONDUCT.md)
+* Transparent - All work is done in public, either from our GitHub repositories or from public meetings.
+
+# Projects
+
+The Confidential Containers organization is composed of multiple projects.
+
+A project is the primary unit of collaboration, therefore each project has its
+own repository and maintainers team. All projects follow the [Code of Conduct](https://github.com/confidential-containers/community/blob/main/CODE_OF_CONDUCT.md).
+
+# Community Members and Roles
+
+Everyone is welcome to participate to the Confidential Containers projects,
+and there are 3 community member roles:
+
+## Contributor
+
+Anyone that contributed to one of the Confidential Containers projects within the
+last 12 months is a *Contributor*. Any merged Pull Request is considered a valid
+contribution.
+
+Contributions are not limited to code alone. Documentation, tests, tools, project
```

Comment:
Just a knit, maybe re-word to:

**Adding or enhancing** documentation, tests...

Reply:
Fixed, thanks.